service-tests/microbadger.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'microbadger', title: 'MicroBadger' })
+module.exports = t;
+
+t.create('image size without a specified tag')
+  .get('/image-size/_/hello-world.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'image size',
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]B$/)
+  }));
+
+t.create('image size with a specified tag')
+  .get('/image-size/_/httpd/alpine.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'image size',
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]B$/)
+  }));
+
+t.create('layers without a specified tag')
+  .get('/layers/_/hello-world.json')
+  .expectJSONTypes(Joi.object().keys({ name: 'layers',
+    value: Joi.number().integer()
```

Comment:
You could probably make this slightly more restrictive by using `Joi.number().integer().positive()`. Also, I believe the formatting is not entirely consistent with your other tests.